Observer/OfflinePaymentObserver.php

```diff
@@ -39,32 +59,42 @@ public function execute(Observer $observer)
     {
         $order   = $observer->getEvent()->getOrder();
         $payment = $order->getPayment();
-
-        if ($payment->getAdditionalData('payment_type') !== 'bill_payment_tesco_lotus') {
-            return $this;
+        $emailData     = new \Magento\Framework\DataObject();
+        switch($payment->getAdditionalInformation('payment_type')) {
+            case 'bill_payment_tesco_lotus':
+                // make sure timezone is Thailand.
+                date_default_timezone_set("Asia/Bangkok");
+                $barcodeHtml   = $this->_helper->convertTescoSVGCodeToHTML($payment->getAdditionalInformation('barcode'));
+                $emailTemplate = 'send_email_tesco_template';
+                break;
+            case 'paynow':
+                $charge_id = $payment->getAdditionalInformation('charge_id');
+                $charge = $this->charge->find($charge_id);
+                date_default_timezone_set("Asia/Singapore");
+                $barcodeHtml   = "<img src= '".$charge->source['scannable_code']['image']['download_uri']."'/>";
+                $emailTemplate = 'send_email_paynow_template';
+                $emailData->setData(['banksUrl' => $this->_assetRepo->getUrl('Omise_Payment::images/supportedbanks.png')]);
```

Comment:
When the supported banks are updated, will we need to update this image as well 🤔?

Can we load from CDN?
If we listed as text, it would at least make it easier to update.

Reply:
not sure how much feasible it would be to load it from CDN. We haven't loaded any of image assets from CDN.
Agreed that adding textual list will be easy to update but the bank names with logos looks great as per UX perspective (I think 🤔 ).